documentation/src/docs/asciidoc/release-notes/release-notes-5.3.0-M1.adoc

```diff
@@ -29,7 +29,9 @@ on GitHub.
 
 * New overloaded variant of `isAnnotated()` in `AnnotationSupport` that accepts
   `Optional<? extends AnnotatedElement>` instead of `AnnotatedElement`.
-
+* New integration test sub-project `platform-tooling-support-tests` added. Set system
+  property `platform.tooling.support.tests.enabled` to `true` to activate it in the
+  main build process.
```

Comment:
We usually don't include non user facing changes in the release notes.

Reply:
+1

Reply:
I'll remove 'em.